plugins/MauticCrmBundle/Tests/Api/Zoho/Xml/WriterTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * @copyright   2017 Mautic Contributors. All rights reserved
+ * @author      Mautic, Inc.
+ *
+ * @link        https://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace MauticPlugin\MauticCrmBundle\Tests\Api\Zoho\Xml;
+
+use MauticPlugin\MauticCrmBundle\Api\Zoho\Xml\Writer;
+
+class WriterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @testdox Test that a single row is generated correctly with numerical index
+     *
+     * @covers  \Writer::row()
```

Comment:
There has to be a full path `\MauticPlugin\MauticCrmBundle\Api\Zoho\Xml\Writer`. CS fixer adds a slash here :(